packages/devextreme/js/__internal/ui/file_uploader/file_uploader.ts

```diff
@@ -177,11 +180,20 @@ interface FileUploaderChunkUploadResponse {
   error?: string;
 }
 
+type CancelButtonClickEvent = NativeEventInfo<KeyboardEvent | MouseEvent | PointerEvent> & {
```

Comment:
Can we use `InteractionEvent type` here? It's a new utility type added [here](https://github.com/DevExpress/DevExtreme/blob/25_2/packages/devextreme/js/events/events.types.d.ts#L119).

```suggestion
type CancelButtonClickEvent = NativeEventInfo<InteractionEvent> & {
```